harvard-lts/dais-integration-tests
# DAIS end-to-end etd int test

**Adds ETD int test to DAIS int-tests**
This avoids the need to involve the ETD pipeline for DAIS testing
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/LTSK8S-903

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
A brief description of what the intended result of the PR will be and/or what problem it solves.

# How should this be tested?
Already deployed to dev
Use the link https://ltsds-cloud-dev-1.lib.harvard.edu:10582/etd/testbatch and verify that a load report appears in the etdadm dropbox OR that a load report comes to the DTS email.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? This is an int test
